lib/fastlane/actions/trello.rb

```diff
@@ -0,0 +1,120 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      TRELLO_WHAT_TO_TEST = :TRELLO_WHAT_TO_TEST      
+    end
+
+    class TrelloAction < Action
+      
+      def self.run(params)
+
+        require 'trello'
+        require 'shellwords'
+        
+        Trello.configure do |config|
+          config.developer_public_key = params[:public_key]
+          config.member_token = params[:member_token]
+        end
+  
+        folder = '.' #subfolder folder is the default folder
+
+        command_prefix = [
+            'cd',
+            File.expand_path(folder).shellescape,
+            '&&'
+        ].join(' ')
+
+        begin
+#          Actions.lane_context[SharedValues::WHAT_TO_TEST] = ""
+          current_version = `#{command_prefix} agvtool what-marketing-version -terse1`.split("\n").last || ''
+          build_number = `#{command_prefix} agvtool what-version`.split("\n").last.to_i
+          title = "Version #{current_version} (#{build_number})"
+
+          board = Trello::Board.find(params[:board_source]) 
+  
+          list = board.lists.select{ |l| l.name == 'Done'}.first
```

Comment:
Maybe the documentation should mention that the column needs to be named `Done`?